sdk/iot/Azure.Iot.Hub.Service/src/Customized/Models/DesiredProperty.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Iot.Hub.Service.Models
+{
+    /// <summary>
+    /// The desired property of a digital twin.
+    /// </summary>
+    [CodeGenModel("Desired")]
+    public partial class DesiredProperty
```

Comment:
This is a PnP property, right? Then we should just mark it as internal and hide it.

Reply:
yes. I mentioned in my description that i'll mark all the pnp related stuff as internal in the next PR.